poolakey/src/main/java/ir/cafebazaar/poolakey/billing/connection/ServiceBillingConnection.kt

```diff
@@ -73,7 +75,12 @@ internal class ServiceBillingConnection(
         }
             .takeIf(
                 thisIsTrue = {
-                    context.packageManager.queryIntentServices(it, 0).isNullOrEmpty().not()
+                    context.packageManager.queryIntentServices(it, 0).isNotEmpty() ||
+                            Build.VERSION.SDK_INT >= Build.VERSION_CODES.N &&
```

Comment:
I recommend moving the additional logic added in these changes to another function, such as a simple function or variable. Then, you can use it here.

Reply:
Done